packages/astro/src/content/loaders/glob.ts

```diff
@@ -190,14 +190,51 @@ export function glob(globOptions: GlobOptions): Loader {
 			}
 
 			const limit = pLimit(10);
+			const skippedFiles: Array<string> = [];
+
+			const contentDir = new URL('content/', settings.config.srcDir);
+
+			function isInContentDir(file: string) {
+				const fileUrl = new URL(file, baseDir);
+				return fileUrl.href.startsWith(contentDir.href);
+			}
+
+			function isConfigFile(file: string) {
+				const configFiles = ['config.js', 'config.ts', 'config.mjs'];
+				const fileUrl = new URL(file, baseDir);
+				return configFiles.some(
+					(configFile) => new URL(configFile, contentDir).href === fileUrl.href
```

Comment:
Could we hoist the `configFiles` beside `contentDir` and pre-compute their `URL`? I'm a bit worried that otherwise there's perf issues.

Reply:
Actually, that means I can use a `Set` instead of `some()` too.